tests/robot-tests/tests/admin_and_public/bau/archive_publication.robot

```diff
@@ -315,6 +321,8 @@ Check that superseded warning link takes user to superseding-publication data-ca
     user checks summary list contains    Publication    ${PUBLICATION_NAME_ARCHIVE}
 
     user waits until page contains    Choose a release
+
+    # failing here
```

Comment:
Has this comment been left in on purpose?

Reply:
Yes, just to denote that the test specifically fails on this line